roles/htcondor/tasks/configure_htcondor.yaml

```diff
@@ -0,0 +1,124 @@
+# configure_htcondor.yaml
+
+- name: install required python packages
+  yum:
+    name: firewalld
```

Comment:
Anything related to firewall install should be in the machine setup part, not in HTCondor's playbook.

Reply:
I think this was already removed if I'm not mistaken